tools/site_and_regional/create_PLUMBER2_sites_csv.ncl

```diff
@@ -0,0 +1,202 @@
+; NCL script
+; create_PLUMBER2_sites_csv.ncl ; Keith Oleson, Sep 2023
+; This script generates a csv file for the NEON site_and_regional infrastructure
+; PLUMBER2_sites.csv
+;**************************************
+
+load "$NCARG_ROOT/lib/ncarg/nclscripts/csm/shea_util.ncl"
+
+begin
+
+  print ("=========================================")
+  print ("Start Time: "+systemfunc("date") )
+  print ("=========================================")
+
+
+; Sites (170 sites)
+  sim = systemfunc("ls -1 " + "/glade/work/oleson/PLUMBER2/datm_files/")
+
+  nsim = dimsizes(sim)
+  site_num       = new(nsim,"integer")
+  atm_ncpl       = new(nsim,"string")
+  startyear      = new(nsim,"string")
+  endyear        = new(nsim,"string")
+  localstartyear = new(nsim,"string")
+  startmonth     = new(nsim,"string")
+  startday       = new(nsim,"string")
+  run_startdate  = new(nsim,"string")
+  starttod       = new(nsim,"string")
+  nsteps         = new(nsim,"string")
+  latitude       = new(nsim,"double")
+  longitude      = new(nsim,"double")
+
+  npfts_ch = 17  ; Number of pfts for canopy top/bottom height
+
+  pfttype = new((/nsim,2/),"integer")  ; pft type
+  pftperc = new((/nsim,2/),"double")   ; pft %
+  pftcth  = new((/nsim,2/),"double")   ; pft canopy top height
+  pftcbh  = new((/nsim,2/),"double")   ; pft canopy bottom height
+; pftcth  = new((/nsim,npfts_ch/),"double")   ; pft canopy top height
```

Comment:
My recollection is that since the 170 PLUMBER2 sites have at most 2 pfts, we (or perhaps just I) decided not to add this information since it would be adding 30 or some-odd columns of information to the csv file that wouldn't be used.

Reply:
OK, we should just remove this then.

Reply:
Removed